json.go

```diff
@@ -52,6 +52,17 @@ func ErrorResponse(err error) JSONResponse {
 	return MessageResponse(500, err.Error())
 }
 
+// Error Responses in the standard Matrix Error format (errcode / error)
```

Comment:
Ideally the first sentence should be a line that starts with `MatrixErrorResponse is ...`.